main.py

```diff
@@ -148,6 +174,12 @@ def main():
     debug = args.get('debug', False)
     port = args.get('port', 8088)
     host = args.get('host', '127.0.0.1')
```

Comment:
Host and port are moved to config file, however here, they are still taken from command line arguments: `--port` and `--host`. Either the config should be removed or these values taken from config. I think config is so far not needed in GrASP and a couple of command line arguments are sufficient.